eth-rpc-adapter/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:14-alpine
+
```

Comment:
is it current latest version of node-alpine image ?

Reply:
latest version is 17 but we use 14 across `bodhi.js` project